pytest/pytest_shim.py

```diff
@@ -0,0 +1,71 @@
+"""A shim for executing pytest that supports test filtering, sharding, and more.
+
+Copied from https://github.com/caseyduquettesc/rules_python_pytest/blob/331e0e511130cf4859b7589a479db6c553974abf/python_pytest/pytest_shim.py
+"""
+
+import sys
+import os
+
+import pytest
+
+
+if __name__ == "__main__":
+    pytest_args = ["--ignore=external"]
```

Comment:
The other template had different args around disabling caching, check if they are needed here.